### DISCUSSION TITLE
Changing name of created branch + combining multiple PRs into one

### DISCUSSION THREAD

**Discussion by Dimmsen** [renovatebot/renovate#20249] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

newest?

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have the following use case. I want to use renovate in a repository. In this repo I want to create a branch (call it WorkingBranch) on which the renovate bot should create the Pull Requests on. The pull requests then should be targetted on to our develop branch and not master branch (call it DevelopBranch). Changing the target branch was possible for me by using "baseBranches". But now I have the problem that renovate will always create a branch called "renovate/..." and create the PR from this branch. 
+ This should instead be the "WorkingBranch" so that I can create this branch with a name I choose in several projects and only work on these branches I choose, not the ones automatically getting created by renovate.
I see the following advantages for this: 1. I can choose a name I want 2. I am not afraid that I have a overflow of created branches from renovate 3. I have all PRs always coming from the same branch which I created
+ Plus is it possible to combine all dependency updates into one PR? I tried packageRules and matchUpdateTypes but it still created- an extra PR for the major updates and one PR for all the other updates.

### What you tried so far.

Couldn't find anything for this to work as intendent

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
`separateMajorMinor` ?

https://docs.renovatebot.com/configuration-options/#separatemajorminor

**Reply by Dimmsen** [reply]
Yes this helped for combining the PRs, thank you.

**Reply by Dimmsen** [reply]
Plus: what i was also searching for my first question was "branchName", which allows to set a name for a branch from which the PR will be created on
